### DISCUSSION TITLE
markdown `==` highlighting works in preview but not in actual page

### DISCUSSION THREAD

**Discussion by LeanderBlume** [requarks/wiki#6993] [Error / Bug Report] [answered]
There is some extended markdown syntax to higlight text by surrounding it with `==`, e.g.

```
here it comes: ==highlight this== but not this
```

This seems to work in the preview window when editing the page, but not in the final rendered page. Is this feature supported or not?

**Comment by NGPixel** [comment, marked as answer]
Use `<mark>highlight this</mark>` instead.
